.circleci/config.yml

```diff
@@ -232,7 +242,30 @@ jobs:
             gem install --user-install dpl
             PATH=$HOME/.gem/ruby/2.3.0/bin:$PATH
             dpl --provider=releases --api-key=${GITHUB_TOKEN} --file=MozillaAppServices.framework.zip --overwrite --skip-cleanup=true
-
+  Deploy website:
+    docker:
+      - image: circleci/node:latest
+    steps:
+      - install-mdbook
+      - checkout
+      - run:
+          name: Build product portal
+          command: |
+            pushd docs/sync-storage-handbook
+            mdbook build
+            popd
+            pushd docs/synconomicon
+            mdbook build
+            popd
+            git config --global user.name "Application Services"
+            git config --global user.email "application-services@mozilla.com"
+            echo "machine github.com login linacambridge password $SITE_DEPLOY_TOKEN" > ~/.netrc
```

Comment:
(I saw we already have `$GITHUB_TOKEN` for Carthage releases...is that yours, @eoger? Would you prefer to reuse it?)

Reply:
Is there a way to install an SSH key on CircleCI instead of echo-ing to a file? It makes me a bit uncomfortable